packages/contentful-migrations/ui/1100000000004-create-m-columns-migration.js

```diff
@@ -82,45 +78,44 @@ module.exports = async function (migration, context) {
   mColumns
     .createField('column_left')
     .name('Column left')
-    .type('Array')
+    .type('Link')
```

Comment:
We could also add a size validation with 1 instead of switching from `Array` to `Link` so the changes would be minimal if it really needs to be kind of stackable (rare edge case ;))

Reply:
I don't know if we really need this back door. But who knows... 🙈